app/Resources/views/themes/tastefulplanet/full/product_list.html.twig

```diff
@@ -0,0 +1,162 @@
+{% extends "@ezdesign/pagelayout.html.twig" %}
+
+{% block page_head %}
+    {% set title = ez_content_name(content) ~ " - " ~ content.contentInfo.publishedDate|localizeddate('short', 'short', app.request.locale) %}
+    {{ parent() }}
+{% endblock %}
+
+{% form_theme form _self %}
+
+{%- block choice_widget_expanded -%}
+    <div {{ block('widget_container_attributes') }}>
+        {%- for child in form %}
+            <div class="element">
+                {{- form_widget(child) -}}
+                {{- form_label(child, null, {translation_domain: choice_translation_domain}) -}}
+            </div>
+        {% endfor -%}
+    </div>
+{%- endblock choice_widget_expanded -%}
+
+{% block content %}
+    {% include '@ezdesign/parts/product_modal_dialog.html.twig' %}
+
+    <section class="content-type-product-list full-view">
+        <div class="banner-image">
+            <div class="thumbnail text-center">
+                {{ ez_render_field(content, 'banner', {
+                    'parameters': {'alias': 'original', 'class': 'img-fluid w-100'}
+                }) }}
+                <div class="title">
+                    <h1 class="field-title">{{ ez_render_field(content, 'description') }}</h1>
+                </div>
+            </div>
+        </div>
+
+        <div class="container product-container">
+            <div class="col-md-12 section-spacer">
+                <span>{{ 'What is on our menu'|trans }}</span>
+            </div>
+
+            <div class="row">
+                <div class="col-md-10 offset-md-1">
+                    {{ form_start(form, { 'action': path(location) }) }}
+
+                    <div class="col-md-12 section-search">
+                        <div class="input-group">
+                            {{ form_widget(form.searchText, {'attr': {'class': 'search', 'placeholder': 'Search Recipes...'}}) }}
```

Comment:
Is there a way to make this placeholder translatable?

Reply:
it is translatable already